README.md

```diff
@@ -438,8 +438,12 @@ More documentation is available for both users and developers.
 To learn about the development goals of mlpack in the short- and medium-term
 future, see the [vision document](https://www.mlpack.org/papers/vision.pdf).
 
+<a href="https://github.com/mlpack/github-readme-stats"><img align="center" src="https://github-readme-stats.vercel.app/api?username=mlpack&show_icons=true&include_all_commits=true&theme=buefy&hide_border=true" alt="mlpack's github stats" /></a> 
```

Comment:
The link doesn't work for me.

Reply:
why

Reply:
Hey @manan-dude, I think the api you are using is not defined for organisations. For example, `https://github-readme-stats.vercel.app/api?username=manan-dude` is working but not `https://github-readme-stats.vercel.app/api?username=mlpack`. A [pull request](https://github.com/anuraghazra/github-readme-stats/pull/2459) has been submitted for the changes, but there is no guarantee of when it will be merged into the main codebase.

Reply:
So what will you suggest

Reply:
I think we have to leave this idea for now.